website/content/en/preview/contributing/working-group.md

```diff
@@ -1,13 +1,69 @@
 description: >
-  Join the Karpenter Working Group
+  Join the Karpenter Community Meetings
 ---
 
-Karpenter's community is open to everyone.
+Karpenter hosts two types of regular community meetings to foster collaboration and maintain project health:
+
+The **Working Group** meetings focus on project direction, technical discussions, and feature development. These bi-weekly sessions bring together contributors, users, and maintainers to review designs, plan releases, and address architectural decisions that shape Karpenter's future.
+
+The **Issue Triage** meetings, held weekly, are dedicated to maintaining project health through focused issue management across our repositories. These sessions ensure proper prioritization, assignment coordination, and timely response to community contributions.
+
+Meeting times alternate to accommodate our global community. All interested participants are welcome to join, contribute, and learn more about Karpenter.
+
+### Issue Triage Meetings
+Karpenter is expanding our community meetings to include dedicated Issue Triage sessions alongside our regular Working Group meetings. This addition aims to improve issue management across both kubernetes-sigs/karpenter and karpenter-provider-aws repositories.
+
+These weekly meetings rotate between repositories and time zones to accommodate our global community. These dedicated triage sessions focus on:
+- Issue prioritization
+- Assignment coordination
+- Discussion of complex issues
+- Maintaining repository health
+
+#### Schedule (Mondays):
+- **Week 1**: kubernetes-sigs/karpenter @ 9:00 AM PST (Europe-friendly)
+- **Week 2**: karpenter-provider-aws @ 9:00 AM PST (Europe-friendly)
+- **Week 3**: kubernetes-sigs/karpenter @ 3:05 PM PST (Americas-friendly)
+- **Week 4**: karpenter-provider-aws @ 3:05 PM PST (Americas-friendly)
+
+### Working Group Meetings
+
+The Karpenter Working Group meetings serve as a collaborative forum for project development, technical discussions, and community engagement. These sessions bring together contributors, users, and maintainers to shape the project's direction and ensure its continued success. Through these meetings, we align on architectural decisions, review critical features, and address community needs.
+
+Key focus areas include:
+- Review and discussion of design proposals and technical implementations
+- Planning of upcoming releases and feature roadmaps
+- Coordination between contributors and cross-team collaboration
+- Technical deep-dives on specific components and challenges
+- Discussion of user feedback and community requirements
+
+The working group provides a platform for both synchronous decision-making and asynchronous follow-ups, ensuring the project maintains its technical excellence while growing its community impact.
+
+
+#### Schedule (Alternating Thursdays):
+
+Our established Working Group meetings continue on a bi-weekly schedule:
+
+- 2:00 PM PST (Americas-friendly)
+- 9:00 AM PST (Europe-friendly)
```

Comment:
There isn't a great India-friendly time but this is probably the most India-friendly time -- the other one sort-of falls earlier in the morning